UCLA Radio/UCLA Radio/data/RadioAPI.swift

```diff
@@ -87,7 +87,8 @@ class RadioAPI {
                 delegate?.failedToFetchData(error)
         }
     }
-    
+
+    static let updatedGiveawaysNotificationName = NSNotification.Name(rawValue: "DidFetchUpdatedTicketGiveaways")
```

Comment:
This applies to all other [bridged foundation types](https://developer.apple.com/library/content/documentation/Swift/Conceptual/BuildingCocoaApps/WorkingWithCocoaDataTypes.html#//apple_ref/doc/uid/TP40014216-CH6) too!